### ISSUE TITLE
iotlabclient is not found on pypi.org

### ISSUE THREAD

**Issue by yatch** [iot-lab/iot-lab-client#2]
I think it was there a while ago; but it's gone...? The following URL is what I'm navigated by clicking "pypi" badge on README.md:

https://pypi.org/project/iotlabclient/

**Comment by rienafairefr** [comment]
Hi @yatch
this project is still in a alpha/beta phase, not really ready for primetime. Once it is we will make an annoucement, add it to the official tools on the website and everything, and of course publish this on PyPi :+1: I published a release at some point on pypi.org but removed it.
I'm curious how you found it and, if you used it, whether you have any feedback as a user ?

Matthieu Berthomé
FIT IoT-LAB team

**Comment by yatch** [comment]
Thank you, @rienafairefr, for your prompt response.

> I'm curious how you found it 

That was about one month ago.

At the beginning, I started using `iotlabcli` as a Python package (library) to write a script. And then, returned values from one of its APIs were not compatible with what is described in [the REST API document](https://api.iot-lab.info). If I remember it correctly, I wanted to have `"scheduled_date"` of a submitted job, which is not obtained using `iotlabcli`.

Then, I read the code, saw `iotlabcli` depends on a set of old APIs, and I thought there should be a library for the latest APIs. Otherwise, I would write one by myself which access the REST APIs directly. I scanned the repository list of `iot-lab`, found this, and found the package on PyPI as well.

> and, if you used it, whether you have any feedback as a user ?

It works very well; no issue so far. I'm satisfied with this package. Feeling thankful to you, FIT IoT-LAB team, and contributors.

I still use `iotlabcli` by the way, for `auth` module to generate `.iotlabrc` if it's missing.
